diff --git a/test-debian-distro-info b/test-debian-distro-info
index 66e1fc3..7b2bab1 100755
--- distro-info/test-debian-distro-info
+++ distro-info/test-debian-distro-info
@@ -18,7 +18,7 @@ COMMAND="${COMMAND:-${0%/*}/debian-distro-info}"
 
 . "${0%/*}/shunit2-helper-functions.sh"
 
-testAlias() {
+dont_testAlias() {
     success "--alias sid" "unstable"
     success "--date 2010-10-20 --alias squeeze" "testing"
     success "--date 2011-11-20 --alias squeeze" "stable"
